src/config.c

```diff
@@ -234,39 +188,23 @@ CONFIG_GETTER(getGcScanSize) {
 
 // MIN_PHONETIC_TERM_LEN
 CONFIG_SETTER(setForkGcInterval) {
-  long long val;
-  if (readLongLongLimit(argv, argc, offset, &val, 1, LLONG_MAX) != REDISMODULE_OK) {
-    return REDISMODULE_ERR;
-  }
-  config->forkGcRunIntervalSec = val;
-  return REDISMODULE_OK;
+  int acrc = AC_GetSize(ac, &config->forkGcRunIntervalSec, AC_F_GE1);
+  RETURN_STATUS(acrc);
 }
 
 CONFIG_SETTER(setForkGcCleanThreshold) {
-  long long val;
-  if (readLongLongLimit(argv, argc, offset, &val, 0, LLONG_MAX) != REDISMODULE_OK) {
-    return REDISMODULE_ERR;
-  }
-  config->forkGcCleanThreshold = val;
-  return REDISMODULE_OK;
+  int acrc = AC_GetSize(ac, &config->forkGcCleanThreshold, 0);
```

Comment:
Why not AC_F_GE0?

Reply:
because size_t is an unsigned type, GE_0 is implicit